<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CALCULATOR</class>
 <widget class="QMainWindow" name="CALCULATOR">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1102</width>
    <height>635</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CALCULATOR</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="AC">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20%;
background-color: blue;
font-size:25px;
</string>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deg">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>130</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20%;
background-color: gray;</string>
    </property>
    <property name="text">
     <string>^</string>
    </property>
   </widget>
   <widget class="QPushButton" name="div">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>130</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20%;
background-color: gray;</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mul">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>200</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20%;
background-color: gray;</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="minus">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>270</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20%;
background-color: gray;</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="plus">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>340</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20%;
background-color: gray;</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bracket_open">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>130</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20%;
background-color: gray;</string>
    </property>
    <property name="text">
     <string>(</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sin">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>200</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20%;
background-color: gray;</string>
    </property>
    <property name="text">
     <string>sin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="asin">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>270</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20%;
background-color: gray;</string>
    </property>
    <property name="text">
     <string>asin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ln">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>340</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20%;
background-color: gray;</string>
    </property>
    <property name="text">
     <string>ln</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Bracket_close">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>130</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20%;
background-color: gray;</string>
    </property>
    <property name="text">
     <string>)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cos">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>200</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20%;
background-color: gray;</string>
    </property>
    <property name="text">
     <string>cos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="acos">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>270</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20%;
background-color: gray;</string>
    </property>
    <property name="text">
     <string>acos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="log">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>340</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20%;
background-color: gray;</string>
    </property>
    <property name="text">
     <string>log</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mod">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>130</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20%;
background-color: gray;</string>
    </property>
    <property name="text">
     <string>mod</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tan">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>200</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20%;
background-color: gray;</string>
    </property>
    <property name="text">
     <string>tan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="atan">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>270</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20%;
background-color: gray;</string>
    </property>
    <property name="text">
     <string>atan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sqrt">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>340</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20%;
background-color: gray;</string>
    </property>
    <property name="text">
     <string>sqrt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="seven">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 1%;
font-size: 20px</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="eight">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>200</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 1%;
font-size: 20px</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nine">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>200</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 1%;
font-size: 20px</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="six">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>270</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 1%;
font-size: 20px</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="five">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>270</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 1%;
font-size: 20px</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="four">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 1%;
font-size: 20px</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="three">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>340</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 1%;
font-size: 20px</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="two">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>340</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 1%;
font-size: 20px</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="one">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 1%;
font-size: 20px</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zero">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 1%;
font-size: 20px</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dot">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>400</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 1%;
font-size: 20px</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="equale">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>400</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20%;

background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 green, stop: 1 blue);</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Main_Field">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>471</width>
      <height>81</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="x_value">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>410</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:blue</string>
    </property>
    <property name="minimum">
     <double>-1000000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>10000000.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="border">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>471</width>
      <height>5</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 green, stop: 1 blue);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>400</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> border-radius: 10%;
background-color: blue;
font-size: 20px;
border: 3px solid white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="x_place">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>400</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> border-radius: 10%;
background-color: rgb(0, 255, 247);
font-size: 20px;
border: 3px solid white;
color: black;</string>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>400</y>
      <width>16</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>20</y>
      <width>481</width>
      <height>371</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 3px solid blue;
border-radius:25px;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="grafic">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>488</y>
      <width>481</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 30px;
 border-radius: 10%;
background-color: blue;
border: 3px solid white;</string>
    </property>
    <property name="text">
     <string>ПОСТРОИТЬ ГРАФИК</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FORCE">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>488</y>
      <width>481</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 30px;
 border-radius: 10%;
background-color: blue;
border: 3px solid white;</string>
    </property>
    <property name="text">
     <string>ДА ПРЕБУДЕТ С ТОБОЙ СИЛА</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="x_min_value">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>400</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 30px;
 border-radius: 1%;
background-color: blue;
border-bottom: 1px solid white;
border-top: 1px solid white;</string>
    </property>
    <property name="minimum">
     <double>-1000000.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="x_max_value">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>400</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 30px;
 border-radius: 1%;
background-color: blue;
border-bottom: 1px solid white;
border-top: 1px solid white;</string>
    </property>
    <property name="minimum">
     <double>-1000000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>1000000.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="y_min_value">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>400</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 30px;
 border-radius: 1%;
background-color: blue;
border-bottom: 1px solid white;
border-top: 1px solid white;</string>
    </property>
    <property name="minimum">
     <double>-1000000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>1000000.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="y_max_value">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>400</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 30px;
 border-radius: 1%;
background-color: blue;
border-bottom: 1px solid white;
border-top: 1px solid white;</string>
    </property>
    <property name="minimum">
     <double>-1000000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>1000000.000000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="x_place_min">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>460</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 255, 247);
font-size: 15px;
border: 3px solid white;
color: black;</string>
    </property>
    <property name="text">
     <string>x min</string>
    </property>
   </widget>
   <widget class="QPushButton" name="x_place_max">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>460</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 255, 247);
font-size: 15px;
border: 3px solid white;
color: black;</string>
    </property>
    <property name="text">
     <string>x max</string>
    </property>
   </widget>
   <widget class="QPushButton" name="y_place_min">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>460</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 255, 247);
font-size: 15px;
border: 3px solid white;
color: black;</string>
    </property>
    <property name="text">
     <string>y min</string>
    </property>
   </widget>
   <widget class="QPushButton" name="y_place_max">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>460</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 255, 247);
font-size: 15px;
border: 3px solid white;
color: black;</string>
    </property>
    <property name="text">
     <string>y max</string>
    </property>
   </widget>
   <zorder>AC</zorder>
   <zorder>deg</zorder>
   <zorder>div</zorder>
   <zorder>mul</zorder>
   <zorder>minus</zorder>
   <zorder>plus</zorder>
   <zorder>bracket_open</zorder>
   <zorder>sin</zorder>
   <zorder>asin</zorder>
   <zorder>ln</zorder>
   <zorder>Bracket_close</zorder>
   <zorder>cos</zorder>
   <zorder>acos</zorder>
   <zorder>log</zorder>
   <zorder>mod</zorder>
   <zorder>tan</zorder>
   <zorder>atan</zorder>
   <zorder>sqrt</zorder>
   <zorder>seven</zorder>
   <zorder>eight</zorder>
   <zorder>nine</zorder>
   <zorder>six</zorder>
   <zorder>five</zorder>
   <zorder>four</zorder>
   <zorder>three</zorder>
   <zorder>two</zorder>
   <zorder>one</zorder>
   <zorder>zero</zorder>
   <zorder>dot</zorder>
   <zorder>equale</zorder>
   <zorder>Main_Field</zorder>
   <zorder>border</zorder>
   <zorder>label_2</zorder>
   <zorder>x_value</zorder>
   <zorder>x_place</zorder>
   <zorder>label_3</zorder>
   <zorder>widget</zorder>
   <zorder>grafic</zorder>
   <zorder>FORCE</zorder>
   <zorder>x_min_value</zorder>
   <zorder>x_max_value</zorder>
   <zorder>y_min_value</zorder>
   <zorder>y_max_value</zorder>
   <zorder>x_place_min</zorder>
   <zorder>x_place_max</zorder>
   <zorder>y_place_min</zorder>
   <zorder>y_place_max</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1102</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
